src/VisualStudio/Core/Def/Commands.vsct

```diff
@@ -266,6 +266,19 @@
         </Strings>
       </Button>
 
+      <Button guid="guidRoslynGrpId" id="cmdidGoToBase" priority="0x0200" type="Button">
```

Comment:
Any vsct file changes shouldn't be necessary in roslyn once it's implemented on the editor side (I believe it should also apply to all the vsct command translations below too).